docs/rfc/rfc-1.adoc

```diff
@@ -184,22 +184,33 @@ reason), all they need to do is not submit their reveal and wait 30 days.
 
 [[sweeping]]
 ==== Sweeping
+
+Governable Parameters
+
+- `sweeping_refund_safety_time`: The amount of time prior to when a UTXO
+  becomes eligible where we will not include it in a sweeping transaction.
+- `sweep_period`: The amount of time we wait between sweeps.
+- `sweep_max_btc`: The amount of total bitcoin that will trigger an early sweep.
```

Comment:
What do you mean by the `total bitcoin` here? The sum of all `unsweptBalance`s on a given wallet?
Would we trigger the sweeping in the situation where `sweep_max_btc` is reached collectively by multiple depositors, but, let's say, only one depositor declared `max_fee` high enough to be included in the sweep?

Reply:
> What do you mean by the total bitcoin here? 

The sum of unswept deposits where each unswept deposit is itself higher than the dust threshold. (which only matters to exclude 1 million deposits of 0.0000001 btc each from triggering a sweep). I'll clarify in the RFC!

> Would we trigger the sweeping in the situation where sweep_max_btc is reached collectively by multiple depositors, but, let's say, only one depositor declared max_fee high enough to be included in the sweep?

My gut instinct is yes, but we have room to iterate because this happens off-chain. We don't know which of the deposits is "too low" until we do the bin-packing and then at that point, we're already mid-sweep.

Though, we *could* simulate a bin-packing with an off-chain fee and see if we have enough deposits with high enough fees to make a sweep worth it before we start one 🤔. Then, if we do, we propose that fee on-chain, and that starts the actual bin-packing (which should be at least as good as the simulated bin-packing), and would save the network fees on sweeping overall :tada:

Reply:
I think these parameters are just a suggestion to the off-chain wallet, at least in v1. If clients in the wallet decide to wait for whatever reason when `sweep_max_btc` is reached, I think this is acceptable.

Reply:
yup! It's a way to sync up the clients to all read the same configuration

Reply:
addressed in 2112b02952cd7de5acbd614eb11f53b07c44f01f

Reply:
Could you also add that clarification about "total bitcoin" referring to the sum of the unswept deposits higher than the dust?

Reply:
I clarified that dust is ignored in e703515 and that we're summing up unswept in a9eab39